core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -286,19 +286,24 @@ class BrokerTopicMetrics(name: Option[String], remoteStorageEnabled: Boolean = f
   case class GaugeWrapper(metricType: String) {
     @volatile private var gaugeObject: Gauge[Long] = _
     final private val gaugeLock = new Object
-    final val aggregatedMetric = new AggregatedMetric()
+    final private val metric = new AggregatedMetric()
 
     def gauge(): Gauge[Long] = gaugeLock synchronized {
       if (gaugeObject == null) {
-        gaugeObject = metricsGroup.newGauge(metricType, () => aggregatedMetric.value(), tags)
+        gaugeObject = metricsGroup.newGauge(metricType, () => metric.value(), tags)
       }
       return gaugeObject
     }
 
+    def aggregatedMetric(): AggregatedMetric = {
```

Comment:
Could you please merge `aggregatedMetric` into `gauge`? we can make `gauge` return `metric`

Reply:
Updated it. Thank you.